testing/workflows/components/kfctl_test.jsonnet

```diff
@@ -313,8 +313,30 @@ local deleteStep = if deleteKubeflow then
   }]
 else [];
 
+// Clean up all the permanent storages to avoid accumulated resource
+// consumption in the test project
+local deleteStorageStep = if deleteKubeflow then
+  [{
+    template: buildTemplate(
```

Comment:
I think it would be better to add it to this script
https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/delete_deployment.sh

Possibly guarded by an option on whether or not to delete storage.

Reply:
I thought about the same but saw this
https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/delete_deployment.sh#L3
which looks fine to me. Since we don't provide option in kfctl.sh parameter to create a storage, it's consistent to me that we don't provide option in kfctl.sh delete.

Let me know if you feel strong about it.